test/row_function_multiproc.py

```diff
@@ -25,7 +25,7 @@ def normalizing_wage(n, df_full):
 
 
 def multiproc_normalize_wage(ids, df):
-    n_worker = psutil.cpu_count() - 1
+    n_worker = 8
```

Comment:
revert this to the previous line - we want it to be distributed across all available cores